project/app/index.js

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
 import { render } from 'react-dom';
+import { HashRouter as Router } from 'react-router-dom';
 import 'availity-uikit/scss/_bootstrap.scss';
 import '@availity/yup';
 import './index.scss';
 import App from './App';
+import { StoreProvider } from './stores';
 
-render(<App />, document.querySelector('#root'));
+render(
+  <StoreProvider>
+    <Router>
+      <App />
```

Comment:
Just for the sake of example purposes, because I am sure someone will ask, can you use the `Switch` component and then add `App` as a route with `path="/"`

Reply:
Will do